rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/config/RocketMQAutoConfiguration.java

```diff
@@ -38,18 +38,18 @@
 
 @Configuration
 @EnableConfigurationProperties(RocketMQProperties.class)
-@ConditionalOnProperty(prefix = "spring.rocketmq", value = "nameServer")
+@ConditionalOnProperty(prefix = "spring.rocketmq", value = "name-server")
```

Comment:
cool, we have the similar PR for relaxed name: https://github.com/apache/rocketmq-spring/pull/2

Reply:
Oops, sorry I missed that. Do you want me to revert that change?

Reply:
@snicoll No change, thanks, we will merge it and do some polish in #2 pr :-)